src/ShortcutsManager.h

```diff
@@ -32,16 +32,18 @@ class ShortcutsManager : public QObject {
     Q_OBJECT
 
 public:
-    void registerShortcut(const QString&, QKeySequence*);
+    void registerShortcut(const QString&, const QString&, QKeySequence*);
     QStringListIterator iterator();
     void setShortcut(const QString&, QKeySequence*);
     QKeySequence* getSequence(const QString&);
     QKeySequence* getDefault(const QString&);
+    QString getLabel(const QString& key);
 
 private:
     QList<QString> shortcutKeys;
     QMap<QString, QKeySequence*> shortcuts;
     QMap<QString, QKeySequence*> defaults;
+    QMap<QString, QString> translations;
```

Comment:
Speaking from [previous experience](https://github.com/Mudlet/Mudlet/pull/5541) - could you document what keys & values are storing here exactly?

Reply:
Those are internal, private fields.

Reply:
Sure - but I still have no idea what I'm looking at just by looking at the declaration

Reply:
Neither might you after a year or two... !

Reply:
At translations!

Reply:
I don't follow - just need a simple comment for all the new maps explaining what the keys and values are

Reply:
Here you go.